pkg/operator/controller/canary/daemonset_test.go

```diff
@@ -13,10 +13,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestDesiredCanaryDaemonSet(t *testing.T) {
-	canaryImage := "openshift/hello-openshift:latest"
+// canaryImageName is the ingress-operator image
+var canaryImageName = "openshift/origin-cluster-ingress-operator:latest"
 
-	daemonset := desiredCanaryDaemonSet(canaryImage)
+func TestDesiredCanaryDaemonSet(t *testing.T) {
+	daemonset := desiredCanaryDaemonSet(canaryImageName, controller.CanaryHealthcheckCommand)
```

Comment:
If we are hard coding the canary container command, does it need to be an argument for `desiredCanaryDaemonset`?